roles/edpm_libvirt/defaults/main.yml

```diff
@@ -26,10 +26,27 @@ edpm_libvirt_image_download_delay: 5
 edpm_libvirt_image_download_retries: 5
 
 edpm_libvirt_image: "quay.io/podified-antelope-centos9/openstack-nova-libvirt:current-podified"
-edpm_libvirt_containers:
-  - "libvirt_virtlogd"
-  - "libvirt_virtnodedevd"
-  - "libvirt_virtproxyd"
-  - "libvirt_virtqemud"
-  - "libvirt_virtsecretd"
+edpm_libvirt_services:
+  - libvirtd
+edpm_libvirt_packages:
+  # main libvirt packages
+  - libvirt
+  - libvirt-admin
+  - libvirt-client
+  - libvirt-daemon
+  # qemu pakcages
+  - qemu-kvm
+  - qemu-img
+  # libguestfs is need for config driver and ephermeral disks
+  - libguestfs
+  - libseccomp
+  # swtpm is needed for TPM support
+  - swtpm
+  - swtpm-tools
+  # edk2-ovmf is needed for UEFI support
+  - edk2-ovmf
+  # ceph-common is needed for ceph support to provide the ceph client lib
+  # for qemu
+  - ceph-common
+
 edpm_libvirt_ceph_path: /var/lib/openstack/config/ceph/
```

Comment:
Hopefully libvirt can use this path. If it can't for some reason we can work something out to keep ceph client and ceph server separate, e.g. ceph server can use `/etc/ceph/$FSID`.